pkg/channel/consolidated/reconciler/controller/controller.go

```diff
@@ -157,9 +157,18 @@ func NewController(
 			DeleteFunc: func(obj interface{}) {
 				pod, ok := obj.(*corev1.Pod)
 				if ok && pod != nil {
-					logger.Debugw("Dispatcher pod deleted. Canceling pod probing.",
+					logger.Debugw("Dispatcher pod deleted. Refreshing pod probing.",
```

Comment:
If these two functions are essentially copy and pasted (save for a debug line).  Abstract the identical code into a different function and pass it with the debug string.

Reply:
I agree here, that `DeleteFunc` and `Addfunc` are very very identical - let's not start copy/paste here for no reason